inst/rmd/qc_report.rmd

```diff
@@ -431,136 +470,13 @@ if(has_filtered & has_processed){
 }
 ```
 
-
-
-## Dimensionality Reduction 
-
-```{r message=FALSE}
-if(has_processed){
-  
-  # use to check if variable genes are present, need for both UMAPs
-  has_var_genes <- !is.null(processed_meta$highly_variable_genes)
-
-  # check that processed SCE contains UMAP and PCA
-  if(!"UMAP" %in% reducedDimNames(processed_sce)){
-    if(!"PCA" %in% reducedDimNames(processed_sce)){
-      
-      # if PCA is missing, check for HVG, if no HVG use all genes 
-      if(has_var_genes){
-        subset_genes <- processed_meta$highly_variable_genes
-      }else{
-        subset_genes <- rownames(processed_sce)
-      }
-      processed_sce <- scater::runPCA(processed_sce,
-                                      subset_row = subset_genes)
-    }
-    
-    processed_sce <- scater::runUMAP(processed_sce, 
-                                     dimred = "PCA")
-  }
-  
-  # create UMAP colored by number of genes detected 
-  scater::plotUMAP(processed_sce,
-                   point_size = 0.3,
-                   point_alpha = 0.5,
-                   colour_by = "detected") +
-    scale_color_viridis_c()+
-    theme_bw() + 
-    guides(
-      color = guide_colorbar(title = "Number of \ngenes detected")
-    )
-  
-} else {
-  # if processed is missing, set to null to skip next UMAPs
-  has_var_genes <- FALSE
-}
-```
-
 Compromised cells, calculated using `miQC` described above, are filtered and removed from the library prior to downstream analyses. 
 The raw counts from all remaining cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 
```

Comment:
I started with editing these sentences to refer to the fact that miQC is not always done, but I went back to looking at it as a whole, and I think we can probably just drop these two sentences? Back on line 426, we say "The below plot highlights cells that were removed prior to normalization and dimensionality reduction." So maybe that covers it?


```suggestion
The raw counts from all cells that remain after filtering low quality cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 
```